.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
 	"root": true,
-	"extends": "@aduth/eslint-config/es5",
```

Comment:
I was trying to find some issue with the ESlint config, and this was changed and I didn't change it back, I'll fix it.

Reply:
Okay, so at the moment I changed this all to be an ES Module, so we wouldn't want es5 right? There is a CommonJS build artifact for supporting that use case, does that create a breaking change?

Reply:
Historically I'd sometimes made a point to target ES5 with the exception of ESM import/export, since most bundlers would handle the import semantics, but wanting broad compatibility for the rest of the code ([old Rollup docs used to suggest this](https://github.com/rollup/rollup/wiki/pkg.module#wait-it-just-means-import-and-export--not-other-future-javascript-features)). But I think it's safe to say that IE11 is finally dead at this point, so it's probably fine to move on from ES5. I think if we make the other breaking change with dropping CJS output, it could make sense to include a note that official support for ES5 is being dropped as well.